src/panels/lovelace/editor/hui-element-editor.ts

```diff
@@ -194,29 +203,44 @@ export abstract class HuiElementEditor<T> extends LitElement {
                   mode="yaml"
                   autofocus
                   .value=${this.yaml}
-                  .error=${Boolean(this._error)}
+                  .error=${Boolean(this._errors)}
                   .rtl=${computeRTL(this.hass)}
                   @value-changed=${this._handleYAMLChanged}
                   @keydown=${this._ignoreKeydown}
                 ></ha-code-editor>
               </div>
             `}
-        ${this._error
+        ${!this._typeSupported && this.configElementType
```

Comment:
```suggestion
        ${this._guiSupported === false
```

Reply:
Without the check for `configElementType` we would also show the warning if no type is provided. But for that we already have a red error message.

Reply:
But now we also show it before the config element is set? Because `_guiSupported` would be undefined?

Reply:
Now I got your point. Adjusted plus fixed an issue with the "Save" button.